src/SmartFormat/Utilities/PluralRules.cs

```diff
@@ -217,44 +231,44 @@ public static class PluralRules
         if (pluralWordsCount == 2) return value < 2 ? 0 : 1;
 
         if (pluralWordsCount == 3) return GetWordsCount3Value(value);
-
+        
         if (pluralWordsCount == 4) return GetWordsCount4Value(value);
 
         return -1;
     }; // DualFromZeroToTwo: one (n == 0..2 fractionate and n != 2), other
 
     private static int GetWordsCount3Value(decimal n)
     {
-        return n switch {
-            0 => 0,
+        return Math.Abs(n) switch
+        {
+            <=0 => 0,
             > 0 and < 2 => 1,
-            > 2 => 2,
-            _ => -1
+            >= 2 => 2
```

Comment:
Maybe this should be the default instead? `_ => 2`

Reply:
👍